### PR TITLE
[BRO] Implement Fog of War

### PR DESCRIPTION
**Pull request by PurpleCrowbar**

### CODE REVIEW

**Comment by PurpleCrowbar** [comment]
[[Fog of War]]

File: Mage.Sets/src/mage/cards/b/BluntTheAssault.java [hunk 1]
```diff
@@ -18,7 +18,7 @@
  */
 public final class BluntTheAssault extends CardImpl {
 
-    private static final FilterCreaturePermanent filter = new FilterCreaturePermanent("creature on the battlefield");
+    private static final FilterCreaturePermanent filter = new FilterCreaturePermanent("creatures on the battlefield");
```
Comment (weirddan455) [inline Mage.Sets/src/mage/cards/b/BluntTheAssault.java:21]: This was correct before no?  "for each creature on the battlefield".

File: Mage.Sets/src/mage/cards/f/FogOfWar.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+import java.util.UUID;
+
+/**
+ * @author PurpleCrowbar
+ */
+public final class FogOfWar extends CardImpl {
+
+    private static final FilterCreaturePermanent filter = new FilterCreaturePermanent("creatures with power 3 or less");
+    private static final FilterCreaturePermanent filter2 = new FilterCreaturePermanent("creatures on the battlefield");
```
Comment (weirddan455) [inline Mage.Sets/src/mage/cards/f/FogOfWar.java:22]: Same here.  Should be "for each creature" not "creatures" (for filter2.  other filter is fine).

**Review by weirddan455 — CHANGES_REQUESTED** [review]

File: Mage.Sets/src/mage/cards/b/BluntTheAssault.java (hunk 1 above)
Comment (PurpleCrowbar) [reply Mage.Sets/src/mage/cards/b/BluntTheAssault.java:21]: In-game it reads "gains life equal to the number of creatures on the battlefield", previously it read "gains life equal to the number of **creature** on the battlefield", I just fixed the typo. Looking at the cards now, maybe setText should be used to get the "you gain 1 life for each.." text

Comment (weirddan455) [reply Mage.Sets/src/mage/cards/b/BluntTheAssault.java:21]: Hmm... looks like these life gain cards are kind of inconsistent on which wording they use.  Probably just use setText on these for now.

**Pull request merged**